tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -144,13 +144,16 @@ def __init__(self, marionette):
         update = Wait(self.marionette).until(expected.element_present(*self._update_locator))
         Wait(self.marionette).until(lambda m: update.location['y'] == 0 and update.is_displayed())
 
-    def tap_update(self, return_details=True):
+    def tap_update(self, return_details=True, return_class=None):
```

Comment:
Ok, you are right, I tried to do it in a way that no other tests were affected. Should I change all tests calling that function then? 

Sorry for not getting it right the first time.